src/mockServer/Services/ProductService.ts

```diff
@@ -0,0 +1,60 @@
+import { IProduct } from "shared/models";
+
+const images: Record<string, string> = {
+  adidas:
+    "https://1000logos.net/wp-content/uploads/2019/06/Adidas-Logo-1991.jpg",
+  nike: "https://static.nike.com/a/images/f_jpg,q_auto:eco/61b4738b-e1e1-4786-8f6c-26aa0008e80b/swoosh-logo-black.png",
+  puma: "https://1000logos.net/wp-content/uploads/2017/05/PUMA-logo.jpg",
+};
+
+class ProductService {
+  private products: IProduct[] = [
+    {
+      id: "1",
+      price: "100",
+      name: "Air Force",
+      brand: "Nike",
+      image: images["nike"],
+    },
+  ];
+
+  async getAllProducts() {
+    return this.products;
+  }
+
+  async addProduct(product: Omit<IProduct, "id">) {
+    const newProduct: IProduct = {
```

Comment:
All the functionality need to be in the controller implementations

Reply:
why?, its mvc. controller only for request and response handling and all business logic on the model(service)